fastapi_auth_jwt_demo/tests/test_fastapi_auth_jwt_demo.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2023 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import time
+
+import jwt
+
+from odoo import tests
+
+
+@tests.tagged("post_install", "-at_install")
+class TestEndToEnd(tests.HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env.ref(
+            "fastapi_auth_jwt_demo.fastapi_endpoint_auth_jwt_demo"
+        ).action_sync_registry()
```

Comment:
```suggestion
        )._handle_registry_sync()
```

@sbidoul  With this change, your tests should be fixes

Reply:
:facepalm: I updated, but still no good.

Reply:
it's not what I see into the code :smirk: 

On my branch, your endpoint is available (https://github.com/acsone/rest-framework/pull/8)
https://github.com/acsone/rest-framework/actions/runs/5313951641/jobs/9620548736?pr=8#step:8:232

Reply:
Okay. Fortunately I'm not doing anything critical today... just authentication.